styles/styles.css

```diff
@@ -302,6 +302,27 @@ input[type="color"] {
   text-transform: none;
 }
 
+
+
+.button.edit.loading,
```

Comment:
Can you use a single classname instead of 2 ?

Reply:
yep